packages/fxa-auth-server/lib/routes/subscriptions.js

```diff
@@ -193,6 +193,8 @@ class DirectStripeRoutes {
       productName: selectedPlan.product_name,
       planEmailIconURL: planMetadata.emailIconURL,
       planDownloadURL: planMetadata.downloadURL,
+      appStoreLink: planMetadata.appStoreLink,
+      playStoreLink: planMetadata.playStoreLink,
```

Comment:
These are fine to grab from the metadata, and it makes sense to store it within metadata. However, have these new metadata properties been documented anywhere?
Also, I think that it would make sense for it to be stored on the `product` because, I would assume, that regardless of the plan being purchased (i.e. monthly, yearly, etc), the download link would be the same for each offering.

Reply:
I will move to product and document

Reply:
FWIW, I don't think the metadata needs to be moved here - [`planMetadata`](https://github.com/mozilla/fxa/blob/310d55d0bbf6cfa1e96ddfbfddd726e1d29bb9c0/packages/fxa-auth-server/lib/routes/subscriptions.js#L160) is a misnomer. It's derived from *both* plan & product metadata, [with the plan able to override the product](https://github.com/mozilla/fxa/blob/310d55d0bbf6cfa1e96ddfbfddd726e1d29bb9c0/packages/fxa-auth-server/lib/routes/utils/subscriptions.js#L12).